### PR TITLE
moved stuff into varsfiles, made things more standard

### PR DESCRIPTION
**Pull request by timothy-spencer**

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -458,30 +434,24 @@ resources:
...
 - name: common-production
-  type: cg-common
+  type: s3-iam
   source:
     bucket_name: {{private-bucket-production}}
-    secrets_passphrase: {{passphrase-production}}
-    secrets_file: nfs-production.yml
-    bosh_cert: bosh.pem
+    versioned_file: nfs-production-decrypted.yml
     region: {{aws-region}}
```
Comment (jmcarp) [inline ci/pipeline.yml]: Looks like it's called `region_name` in the s3 resource: https://github.com/alphagov/paas-s3-resource#source-configuration

File: bosh/manifest.yml [hunk 2]
```diff
@@ -30,9 +30,9 @@ instance_groups:
   - name: consul_agent
     release: consul
     consumes:
-      consul_common: {from: consul_common_link, deployment: (( param "specify cf deployment" ))}
+      consul_common: ((cf-deployment))
```
Comment (jmcarp) [inline bosh/manifest.yml]: Looks like we're changing this value from

```
{from: consul_common_link, deployment: ((cf_deployment))}
```

to 

```
((cf_deployment))
```

Is that intentional?

File: ci/pipeline.yml (hunk 1 above)
Comment (jmcarp) [reply ci/pipeline.yml]: Also, we should be able to drop `cg-common` from `resource_types`.

File: bosh/manifest.yml (hunk 2 above)
Comment (timothy-spencer) [reply bosh/manifest.yml]: Not sure what you mean there?  The before was `{from: consul_common_link, deployment: (( param "specify cf deployment" ))}`, which is a spruce thing, which we are trying to get rid of, so I replaced it with a varsfile variable.  Is there a better way to do that?

File: ci/pipeline.yml (hunk 1 above)
Comment (timothy-spencer) [reply ci/pipeline.yml]: Ah.  Thanks!  Fixed.

File: bosh/manifest.yml (hunk 2 above)
Comment (jmcarp) [reply bosh/manifest.yml]: The `(( param "specify cf deployment" ))` piece uses spruce, but the rest of the line is plain yaml.

Comment (timothy-spencer) [reply bosh/manifest.yml]: Huh.  Never seen that before.  Is there some documentation on what that from: stuff is, then?

Comment (jmcarp) [reply bosh/manifest.yml]: https://bosh.io/docs/links/

File: bosh/secrets.example.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# These are secrets because the volume names need to be secret,
+# or a customer will see this and be able to mount up another
+# customer's volumes.  These should probably be something like a uuid.
+nfstestserver-volumes:
+  - XXX
+
+# we are making this live here because it seems like a good idea to not
+# publicize customer names.  This is a list of cf orgs that have this
+# service enabled for it.
+service-organization-development: XXX
```
Comment (jmcarp) [inline bosh/secrets.example.yml]: This isn't a bosh secret.

File: bosh/secrets.example.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# These are secrets because the volume names need to be secret,
+# or a customer will see this and be able to mount up another
+# customer's volumes.  These should probably be something like a uuid.
```
Comment (jmcarp) [inline bosh/secrets.example.yml]: How about dropping the "probably": "Volume names should be UUIDs."

Comment (timothy-spencer) [reply bosh/secrets.example.yml]: Good call.

File: bosh/secrets.example.yml (hunk 3 above)
Comment (timothy-spencer) [reply bosh/secrets.example.yml]: Bah.  How did I not see that!  :-)

**Comment by timothy-spencer** [comment]
This has been deployed into development and tested itself out as OK.  Once it gets into master, I'll change the branch back in the pipeline and let it go through staging/prod.

File: ci/secrets.example.yml [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+production-auth-name: XXX
+production-auth-pass: XXX
+
+slack-channel: XXX
+slack-icon-url: XXX
+slack-news-channel: XXX
+slack-username: XXX
+slack-webhook-url: XXX
+
+passphrase-development: XXX
```
Comment (jmcarp) [inline ci/secrets.example.yml]: Looks like these aren't used.

Comment (timothy-spencer) [reply ci/secrets.example.yml]: Removed!

**Review by jmcarp — APPROVED** [review]

**Pull request merged**